platforms/corda-ent/charts/idman/templates/deployment.yaml

```diff
@@ -0,0 +1,250 @@
+                    echo "${subordinate_crl}" | base64 -d > ${MOUNT_PATH}/crl-files/subordinate.crl
+
+                    echo "Successfully got crl certifcates"
+                    break
+                  fi 
+                  COUNTER=`expr "$COUNTER" + 1`
+              done
+
+              if [ "$COUNTER" -ge {{ $.Values.healthcheck.readinessthreshold }} ]
+              then
+                # printing number of trial done before giving up
+                echo "$COUNTER"
+                echo "crl certificates might not have been put in vault. Giving up!!!"
+                exit 1
+              fi
+              echo "Done"
+        volumeMounts:
+        - name: certificates
+          mountPath: /DATA            
+      containers:
+      - name: idman
+        image: "{{ required "idman[main]: missing value for .Values.dockerImage.name" .Values.dockerImage.name }}:{{ required "idman[main]: missing value for .Values.dockerImage.tag" .Values.dockerImage.tag }}"
+        imagePullPolicy: {{ .Values.dockerImage.pullPolicy }}
+        env:
+          - name: ACCEPT_LICENSE
+            value: "{{required "You must accept the license agreement to run this software" .Values.acceptLicense }}"
+        command: ["/bin/bash", "-c"]
+        args:
+        - |-
+          cp CM/run.sh bin/
+          chmod +x bin/*
+          bin/run.sh
+        volumeMounts:
+        - name: idman-etc
+          mountPath: /opt/corda/etc
+        - name: idman-conf
+          mountPath: /opt/corda/CM/run.sh
+          subPath: run.sh
+        - name: idman-logs
+          mountPath: /opt/corda/logs
+        - name: idman-h2
+          mountPath: /opt/corda/h2
+        - name: certificates
+          mountPath: /opt/corda/DATA
+        resources:
+          requests:
+            memory: {{ .Values.cordaJarMx }}G
+          limits:
+            memory: {{ add .Values.cordaJarMx 2 }}G
+      - name: logs
+        image: "{{ required "idman[logs]: missing value for .Values.dockerImage.name" .Values.dockerImage.name }}:{{ required "idman[logs]: missing value for .Values.dockerImage.tag" .Values.dockerImage.tag }}"
+        imagePullPolicy: {{ .Values.dockerImage.pullPolicy }}
+        env:
+        - name: ACCEPT_LICENSE
+          value: "{{required "You must accept the license agreement to run this software" .Values.acceptLicense }}"
+        command: ["/bin/bash", "-c"]
+        args:
+        - |-
+          cd /opt/corda
+          while true; do tail -f logs/identity-manager/*.log 2>/dev/null; sleep 5; done
+          # in case sth went wrong just wait indefinitely ...
+          tail -f /dev/null
+        volumeMounts:
+        - name: idman-logs
+          mountPath: /opt/corda/logs
+      imagePullSecrets:
+      - name: {{ .Values.dockerImage.imagePullSecret }}
+      volumes:
+        - name: idman-conf
+          configMap:
+            name: {{ .Values.nodeName }}-conf
+        - name: idman-etc
```

Comment:
etc is storing in memory and not PVC. Have you tested the idman pod comes up with exact same configurations when deleted?

Reply:
Yes I have tested it.